openlibrary/plugins/upstream/forms.py

```diff
@@ -113,7 +111,13 @@ class RegisterForm(Form):
         Checkbox(
             'ia_newsletter',
             description=_(
-                """I want to receive news, announcements, and resources from the <a href="https://archive.org/">Internet Archive</a>, the non-profit that runs Open Library."""
+                """Best of the archive : useful resources, unique stories, and fun find from our collections."""
+            ),
+        ),
+        Checkbox(
+            'ib_newsletter',
```

Comment:
```suggestion
            'ia_events_notifications',
```
Not sure what `ib` stands for.  Let's call this `ia_events_notifications`.